core/src/com/po/fuck/view/classdrawers/factories/RoomDrawerFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.po.fuck.view.classdrawers.factories;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.po.fuck.view.CenterDrawer;
+import com.po.fuck.view.Renderer;
+import com.po.fuck.view.classdrawers.ClassDrawer;
+import com.po.fuck.model.Room;
+
+public class RoomDrawerFactory {
+    
+    static {
+        Renderer.addDrawer(Room.class, RoomDrawerFactory.get(Room.class, new Sprite(new Texture("island2.png"))));
+    }
+
+    @SuppressWarnings("all")
+    public static <T extends Room> ClassDrawer<T> get(Class<T> clz, Sprite other_sprite){
```

Comment:
`other_sprite` - not java style naming, also not very obvious, maybe add javadoc or rename it